CompuCell3D/core/Demos/CompuCellPythonTutorial/ElongatedCellsTutorial/Tutorial_09/README.txt

```diff
@@ -0,0 +1 @@
+This demo shows how to create in Python multiple cells clusters  We are also applying a random force to the cells that "cap" the cluster
```

Comment:
nit: Add punctuation
It may also be worth mentioning that this is part 9 of 10 for the Elongated Cells walk through, in case it is not already obvious by the name.